payouts.go

```diff
@@ -0,0 +1,95 @@
+package goshopify
+
+import (
+	"fmt"
+
+	"github.com/shopspring/decimal"
+)
+
+const payoutsBasePath = "shopify_payments/payouts"
+
+// PayoutService is an interface for interfacing with the payout endpoints of
```

Comment:
The Shopify docs call it `The Payouts resource` so this should be the Payouts service, etc - all plural.